apps/opik-frontend/src/hooks/charts/useChartTickDefaultConfig.ts

```diff
@@ -0,0 +1,121 @@
+import { useCallback, useMemo } from "react";
+import { isInteger, max, min } from "lodash";
```

Comment:
Lets not mix type of import from lodash, lets import utils one by one (theoretically it affects bundle size)